sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -3,15 +3,11 @@
 # Licensed under the MIT License.
 # ------------------------------------
 import json
+from json import JSONDecodeError
```

Comment:
Depending on how the error type is referenced, it may be more convenient to just reference it as `json.JSONDecodeError` and remove this import since we already `import json`